app/src/main/java/org/jellyfin/androidtv/playback/nextup/NextUpActivity.kt

```diff
@@ -50,6 +49,14 @@ class NextUpActivity : FragmentActivity() {
 		}
 	}
 
+	private fun safelyLoadBitmap(url: String): Bitmap? {
+		return try {
```

Comment:
```suggestion
	private fun safelyLoadBitmap(url: String) = try {
```
(also, one of the closing brackets should be removed)